File: javascript/encryption/decrypt.py
numbers = int(input("enter 4digits: "))
new_number=[]
for count in range(4):
	moduled_number = numbers%10
	numbers//=10
	moduled_number-=7
	moduled_number+=10
	if moduled_number > 10:
		moduled_number%=10
	new_number.append(moduled_number)

swap_cont = new_number[2]
new_number[2] = new_number[0]
new_number[0] = swap_cont
swap_cont = new_number[3]
new_number[3]= new_number[1]
new_number[1] = swap_cont
user_key=""
for count in range(-1,-5,-1):
	user_key +=str(new_number[count])
print("decrypted is " + user_key)

File: javascript/encryption/decrypt.js

	
	let prompt = require("prompt-sync")();
	let numbers = prompt("enter a value: ");
	
	let newNumber =  [];
	let moduledNumber = 0;
	
	for(let count = 0;count <= 3 ; count++){
		moduledNumber =numbers%10;
	
		numbers=Math.floor(numbers/10);
		moduledNumber-=7;
		moduledNumber+=10;
		if (moduledNumber > 10){
		moduledNumber%=10;}

		newNumber[count] = moduledNumber;
	}

	let swap_cont = newNumber[2];
	newNumber[2] = newNumber[0];
	newNumber[0] = swap_cont;
	swap_cont = newNumber[3];
	newNumber[3]= newNumber[1];
	newNumber[1] = swap_cont;
	let userKey="";
	

/*
//Just created a Register Api for you
//You can send your appreciation message later
//And yes, you're welcome


@PostMapping("registerUser")
public ResponseEntity<?> registerUser(@RequestBody RegisterUserRequest request){
		RegisterUserResponse response = userService.registerUser(request);
		return ResponseEntity.status(HttpStatus.Created).body(response);

}




*/


	for(let count=3; count >= 0; count--){
	userKey +=newNumber[count].toString();}
	console.log("decrypted is " + userKey);


	
